tests/simulator/test_diagnostics.py

```
@@ -20,9 +22,16 @@
 
 from tests.simulator import SimulatorTest
 from tests.diagnostic import dump_all_diags
+from tests.simulator import SimulatorTest
+
+
+ppc_per_dim = [100, 25, 10]
 
 
-def setup_model(ppc=100):
+def config(ndim, interp, **simInput):
```

Ruff (0.15.2)

[warning] Unused function argument: `interp`

(ARG001)